Another internal webcam causing crashes

From https://discord.com/channels/176186766946992128/528555967827148801/1032810205769113630 - 

```

[2022-10-20 16:57:20] [Camera - VisionSourceManager] [INFO] Adding local video device - "Integrated Webcam" at "\\?\usb#vid_0bda&pid_5510&mi_00#6&212fed36&0&0000#{e5323777-f976-4f5b-9b55-b94699c46e44}\global"
[2022-10-20 16:57:20] [Camera - VisionSourceManager] [DEBUG] Trying to match 1 unmatched configs...
[2022-10-20 16:57:20] [Camera - VisionSourceManager] [DEBUG] Trying to find a match for loaded camera Integrated Webcam with path \\?\usb#vid_0bda&pid_5510&mi_00#6&212fed36&0&0000#{e5323777-f976-4f5b-9b55-b94699c46e44}\global
[2022-10-20 16:57:20] [Camera - VisionSourceManager] [DEBUG] Matched the config for Integrated Webcam to a physical camera!
[2022-10-20 16:57:20] [Camera - VisionSourceManager] [DEBUG] After matching loaded configs 0 cameras were unmatched.
[2022-10-20 16:57:20] [Camera - VisionSourceManager] [DEBUG] Matched or created 1 camera configs!
[2022-10-20 16:57:20] [Camera - VisionSourceManager] [DEBUG] Matched config for camera "USBFrameProvider - opencv_Integrated_Webcam" and loaded 1 pipelines
[2022-10-20 16:57:20] [Camera - VisionSourceManager] [INFO] Adding 1 configs to VMM.
[2022-10-20 16:57:20] [WebServer - DataChangeService] [DEBUG] Added subscriber - Sources: DCS_WEBSOCKET, DCS_HTTP, DCS_NETWORKTABLES, DCS_VISIONMODULE, DCS_OTHER, Destinations: DCD_ACTIVEMODULE, DCD_ACTIVEPIPELINESETTINGS, DCD_GENSETTINGS, DCD_UI, DCD_OTHER
[2022-10-20 16:57:20] [VisionModule - VisionModule - Integrated_Webcam] [INFO] Setting pipeline to 0
[2022-10-20 16:57:20] [VisionModule - PipelineManager] [DEBUG] Creatig Reflective pipeline
[2022-10-20 16:57:20] [Camera - USBCameraSource - Integrated_Webcam] [ERROR] Failed to set camera brightness!
[2022-10-20 16:57:20] [Camera - USBCameraSource - Integrated_Webcam] [ERROR] VideoException [edu.wpi.first.cscore.VideoException: property write failed]
	at edu.wpi.first.cscore.CameraServerJNI.setCameraBrightness(Native Method)
	at edu.wpi.first.cscore.VideoCamera.setBrightness(VideoCamera.java:27)
	at org.photonvision.vision.camera.USBCameraSource$USBCameraSettables.setBrightness(USBCameraSource.java:198)
	at org.photonvision.vision.processes.VisionModule.setPipeline(VisionModule.java:399)
	at org.photonvision.vision.processes.VisionModule.<init>(VisionModule.java:149)
	at org.photonvision.vision.processes.VisionModuleManager.addSources(VisionModuleManager.java:59)
	at org.photonvision.vision.processes.VisionSourceManager.tryMatchUSBCams(VisionSourceManager.java:84)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

[2022-10-20 16:57:20] [Camera - USBCameraSource - Integrated_Webcam] [DEBUG] Setting camera exposure to 100
[2022-10-20 16:57:20] [Camera - USBCameraSource - Integrated_Webcam] [ERROR] Failed to set camera exposure!
[2022-10-20 16:57:20] [Camera - USBCameraSource - Integrated_Webcam] [ERROR] VideoException [edu.wpi.first.cscore.VideoException: property write failed]
	at edu.wpi.first.cscore.CameraServerJNI.setCameraExposureManual(Native Method)
	at edu.wpi.first.cscore.VideoCamera.setExposureManual(VideoCamera.java:74)
	at org.photonvision.vision.camera.USBCameraSource$USBCameraSettables.setExposure(USBCameraSource.java:186)
	at org.photonvision.vision.processes.VisionModule.setPipeline(VisionModule.java:408)
	at org.photonvision.vision.processes.VisionModule.<init>(VisionModule.java:149)
	at org.photonvision.vision.processes.VisionModuleManager.addSources(VisionModuleManager.java:59)
	at org.photonvision.vision.processes.VisionSourceManager.tryMatchUSBCams(VisionSourceManager.java:84)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

Add that PID and VID to the Completely Broken Quirk

--- a/photon-core/src/main/java/org/photonvision/vision/camera/QuirkyCamera.java
+++ b/photon-core/src/main/java/org/photonvision/vision/camera/QuirkyCamera.java
@@ -29,6 +29,8 @@ public class QuirkyCamera {
                             0x5A3,
                             CameraQuirk.CompletelyBroken), // Chris's older generic "Logitec HD Webcam"
                     new QuirkyCamera(0x825, 0x46D, CameraQuirk.CompletelyBroken), // Logitec C270
+                    new QuirkyCamera(0x0bda, 0x5510, CameraQuirk.CompletelyBroken), // A laptop internal camera someone found broken
+                    new QuirkyCamera(0x0, 0x0, CameraQuirk.CompletelyBroken), // SnapCamera on Windows
                     new QuirkyCamera(0x2000, 0x1415, CameraQuirk.Gain, CameraQuirk.FPSCap100), // PS3Eye
                     new QuirkyCamera(
                             -1, -1, "mmal service 16.1", CameraQuirk.PiCam), // PiCam (via V4L2, not zerocopy)

--- a/photon-core/src/main/java/org/photonvision/vision/camera/QuirkyCamera.java
+++ b/photon-core/src/main/java/org/photonvision/vision/camera/QuirkyCamera.java
@@ -29,8 +29,12 @@ public class QuirkyCamera {
                             0x5A3,
                             CameraQuirk.CompletelyBroken), // Chris's older generic "Logitec HD Webcam"
                     new QuirkyCamera(0x825, 0x46D, CameraQuirk.CompletelyBroken), // Logitec C270
-                    new QuirkyCamera(0x0bda, 0x5510, CameraQuirk.CompletelyBroken), // A laptop internal camera someone found broken
-                    new QuirkyCamera(0x0, 0x0, CameraQuirk.CompletelyBroken), // SnapCamera on Windows
+                    new QuirkyCamera(
+                            0x0bda,
+                            0x5510,
+                            CameraQuirk.CompletelyBroken), // A laptop internal camera someone found broken
+                    new QuirkyCamera(
+                            -1, -1, "Snap Camera", CameraQuirk.CompletelyBroken), // SnapCamera on Windows
                     new QuirkyCamera(0x2000, 0x1415, CameraQuirk.Gain, CameraQuirk.FPSCap100), // PS3Eye
                     new QuirkyCamera(
                             -1, -1, "mmal service 16.1", CameraQuirk.PiCam), // PiCam (via V4L2, not zerocopy)

